packages/labs/src/protobufjs/BUILD.bazel

```diff
@@ -69,8 +70,24 @@ nodejs_binary(
     install_source_map_support = False,
 )
 
+# Runtime libraries needed by the protobufjs library.
+# Any JS code produced by the ts_proto_library rule has a runtime dependency on these scripts.
+js_library(
```

Comment:
hmmmmmm.. looks like js_library has the additional amd_names attribute which is being used. will note that in https://github.com/bazelbuild/rules_nodejs/issues/149

Reply:
sigh, yes that needs a new home